clients/http_client.py

```diff
@@ -0,0 +1,28 @@
+import requests
+
+
+class HTTPClient:
+    """
+    HTTP client for jobs.tut.by
```

Comment:
it is not for jobs.tut.by, it could be used for all requests

Reply:
Fixed